.github/workflows/workflow.yaml

```diff
@@ -91,7 +100,9 @@ jobs:
           key: ${{ hashFiles('examples/**', 'proxywasm/**') }}
 
       - name: run e2e test
-        run: CGO_ENABLED=0 go test ./e2e/... -v
+        run: make test.e2e
+        env:
+          CGO_ENABLED: "0"
```

Comment:
I am curious about this. Are we using some sort of CGO?

Reply:
I forgot the reason so I tried https://github.com/tetratelabs/proxy-wasm-go-sdk/runs/4945452344?check_suite_focus=true 

looks like we execute e2e tests on each envoy container where CGO is enabled by default and no gcc is available.  Maybe I should add comment! Thanks